web/source/kmwdomevents.ts

```diff
@@ -209,6 +219,7 @@ namespace com.keyman {
         Ltarg = Ltarg.parentNode as HTMLElement;
       }
 
+      // TODO:  Needs tidy-up.
```

Comment:
This is a note toward #539, as the block below wants to push `Ltarg` back toward the design-mode iframe's `Document` again - something we should not allow, but is beyond the scope of this PR to fix.